### PR TITLE
[MIG] mrp_bom_location: Migration to 12.0

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

**Comment by kittiu** [comment]
@Saran440 please check travis ci and clear error

**Review by LoisRForgeFlow — APPROVED** [review]
Code review and tested on runbot. :+1:

File: mrp_bom_location/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+#. Go to *Manufacturing > Bill of Materials*.
+#. Pick or create one of them.
+#. You will see a new field to fill called "Location".
+#. One the structure report *BOM > Print > BOM Structure* location field is present
```
Comment (tbaden) [inline mrp_bom_location/readme/USAGE.rst]: Typo:
On the structure report *BOM > Print > BOM Structure* location field is present

**Review by tbaden — APPROVED** [review]
code review: lgtm
but found a typo :)

File: mrp_bom_location/readme/USAGE.rst (hunk 1 above)
Comment (Saran440) [reply mrp_bom_location/readme/USAGE.rst]: Fixed it.

**Pull request merged**